code/modules/culture_descriptor/culture/cultures_human.dm

```diff
@@ -5,6 +5,7 @@
 /singleton/cultural_info/culture/human
 	name = CULTURE_HUMAN
 	description = "You are from one of various planetary cultures of humankind."
+	default_language = LANGUAGE_GALCOM
```

Comment:
this shouldn't be a problem. it was like this before Bay fully removed GalCom from the game anyways.

Reply:
this will also fix adminspawned humans not having any language actually